src/components/EventHighlight/EventHighlight.js

```diff
@@ -93,6 +103,16 @@ function EventHighLight({ classes }) {
 				<Typography variant="body1">
 					{event.description}
 				</Typography>
+				<Link href={event.link}
+					target="_blank" rel="noopener noreferrer"
+					underline="none">
```

Comment:
```suggestion
				<Link 
				  href={event.link}
					target="_blank" 
					rel="noopener noreferrer"
					underline="none"
			  >
```
If you were to break up props into the multiple lines, keep only one prop on each line and keep the `>` on a line of its own!